### PR TITLE
CDAP-528: Update MovieRecomender example to read directly from Stream

### PR DESCRIPTION
**Pull request by artemmakogon**
This PR does not contains changes in docs. They will be added in separate PR

### CODE REVIEW

**Comment by artemmakogon** [comment]
@rohitsinha54 please review

File: MovieRecommender/pom.xml [hunk 1]
```diff
@@ -59,6 +60,32 @@
 
   <dependencies>
     <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>${guava.version}</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.hadoop</groupId>
```
Comment (rohitsinha54) [inline MovieRecommender/pom.xml:69]: scope ?

Comment (artemmakogon) [inline MovieRecommender/pom.xml:69]: provided, but scope was written after exclusions. I've change that

**Comment by rohitsinha54** [comment]
@artemmakogon : Have you tested this work with the changes in the program class loader ?

**Comment by artemmakogon** [comment]
@rohitsinha54 Please close this PR. I have created https://github.com/caskdata/cdap-apps/pull/40 against develop

**Pull request closed without merging**